aws/users/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+data "terraform_remote_state" "account_settings" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.namespace}-${var.stage}-terraform-state"
+    key    = "account-settings/terraform.tfstate"
+  }
+}
+
+
+data "terraform_remote_state" "root_iam" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.namespace}-${var.stage}-terraform-state"
+    key    = "root-iam/terraform.tfstate"
+  }
+}
+
+data "aws_iam_account_alias" "default" {}
+
+output "account_alias" {
+  value = "${data.aws_iam_account_alias.default.account_alias}"
+}
+
+locals {
+  signin_url = "${data.terraform_remote_state.account_settings.signin_url}"
```

Comment:
`terraform fmt`

Reply:
locals not in use?

Reply:
are we missing the actual `user` files?

Reply:
See https://github.com/cloudposse/root.cloudposse.co/pull/24